docker/Dockerfile

```diff
@@ -61,6 +61,7 @@ ARG DISABLE_LZMA=OFF
 ARG DISABLE_BZIP2=OFF
 ARG ENABLE_SCRIPTING=ON
```

Comment:
Should we have scripting enabled by default? I just realized this and because the DISABLE_PYTHON_SCRIPTING flag is off by default, it automatically installs python packages which adds 80MBs to the image. I'm not sure if scripting should be enabled. (If it's needed by the users of the published minimal image then it should be enabled in that config)

Reply:
I am not sure about this, the minimal image explicitly disables this though
https://github.com/apache/nifi-minifi-cpp/blob/main/cmake/DockerConfig.cmake#L77-L78

Reply:
In that case as that is our released docker image I think we can disable it by default. It shouldn't have any impact and I don't think it should be included if not specified.

Reply:
I prefer disabled by default. We do the same in cmake because libpython is a problematic dependency for shipping binary packages. It changes soname every minor version, which breaks compatibility with any other version.

Reply:
changed it in https://github.com/apache/nifi-minifi-cpp/pull/1244/commits/cf4fbe4c44e764f775278784a06ee2aef6893b84